iris-mpc-cpu/src/network/grpc.rs

```diff
@@ -0,0 +1,491 @@
+use super::Networking;
+use crate::{
+    execution::{local::get_free_local_addresses, player::Identity},
+    network::SessionId,
+};
+use backoff::{future::retry, ExponentialBackoff};
+use dashmap::DashMap;
+use eyre::{eyre, OptionExt};
+use party_node::{
+    party_node_client::PartyNodeClient,
+    party_node_server::{PartyNode, PartyNodeServer},
+    SendRequest, SendResponse,
+};
+use std::sync::Arc;
+use tonic::{
+    async_trait,
+    transport::{Channel, Server},
+    Request, Response, Status,
+};
+
+mod party_node {
+    tonic::include_proto!("party_node");
+}
```

Comment:
I'm always more inclined to put the generated files in version control to be able to look at them, however, I get that this is the official recommended way of tonic.